bundles/org.openhab.binding.sony/src/main/resources/ESH-INF/thing/scalar.xml

```diff
@@ -196,6 +196,230 @@
+		<label>Storage Mounted</label>
+		<description>The storage mounted status</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="scalarsystemstoragepermission" advanced="true">
+		<item-type>String</item-type>
+		<label>Storage Permission</label>
+		<description>The storage permission status</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="scalarsystemstorageposition" advanced="true">
+		<item-type>String</item-type>
+		<label>Storage Position</label>
+		<description>The position of the storage (front, back, internal, etc)</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="scalarsystemstorageprotocol" advanced="true">
+		<item-type>String</item-type>
+		<label>Storage Protocol</label>
+		<description>The storage protocol</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="scalarsystemstorageregistrationdate" advanced="true">
+		<item-type>String</item-type>
+		<label>Storage Registration Date</label>
+		<description>The storage registration date</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="scalarsystemstoragesystemareacapacitymb" advanced="true">
+		<item-type>Integer</item-type>
+		<label>Storage System Capacity</label>
+		<description>The storage system capacity (in MB)</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="scalarsystemstoragetimesectofinalize" advanced="true">
+		<item-type>Integer</item-type>
+		<label>Storage Time to Finalize</label>
+		<description>The time (in seconds) to finalize</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="scalarsystemstoragetimesectogetcontents" advanced="true">
+		<item-type>Integer</item-type>
+		<label>Storage Time to Get Contents</label>
+		<description>The time (in seconds) to get storage contents</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="scalarsystemstoragetype" advanced="true">
+		<item-type>String</item-type>
+		<label>Storage Type</label>
+		<description>The storage type</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="scalarsystemstorageuri" advanced="true">
+		<item-type>String</item-type>
+		<label>Storage URI</label>
+		<description>The storage uri</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="scalarsystemstorageusbdevicetype" advanced="true">
+		<item-type>String</item-type>
+		<label>Storage USB Type</label>
+		<description>The storage USB device type</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="scalarsystemstoragevolumelabel" advanced="true">
+		<item-type>String</item-type>
+		<label>Storage Label</label>
+		<description>The storage label</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="scalarsystemstoragewholecapacitymb" advanced="true">
+		<item-type>Integer</item-type>
```

Comment:
Is Integer as supported item type? Isn't supposed to be Number?

Reply:
Yep - copy pasted something - should be a number - fixed.